trajectory_manager:
  ros__parameters:
    robot_name: "robot_0"
    waypoints: [0.0, 0.0, 0.0,  # x y z
                5.0, 5.0, 5.0,
                10.0, 0.0, 0.0]
    obstacles: [20.0, 20.0, 20.0]
    obstacle_distance: 1.0
    magic_fabian_constant: 6.0
    time_factor: 1.0



